### PR TITLE
CSPL-1728 : upgrade scenario document changes

### PR DESCRIPTION
**Pull request by vivekr-splunk**
changes are done to upgrade scenario, added 2 section
upgrade from 1.1.0 and later
upgrade before 1.1.0
changed 1.1.0 to 2.0.0
addressed CSPL-1814 changes , added namespace to all the examples

### CODE REVIEW

File: docs/SplunkOperatorUpgrade.md [hunk 1]
```diff
@@ -16,99 +16,134 @@ A Splunk Operator for Kubernetes upgrade might include support for a later versi
...
+```
+kubectl apply -f splunk-operator-namespace.yaml
+```
+​
+After applying the yaml, a new operator pod will be created and the existing operator pod will be terminated. Example:
+​
+```
+kubectl get pods
+NAME                                                  READY   STATUS    RESTARTS   AGE
+splunk-operator--controller-manager-75f5d4d85b-8pshn   1/1     Running   0          5s
```
Comment (gaurav-splunk) [inline docs/SplunkOperatorUpgrade.md:44]: nit: splunk-operator-controller-manager-75f5d4d85b-8pshn

**Review by gaurav-splunk — APPROVED** [review]

File: docs/AppFramework.md [hunk 2]
```diff
@@ -208,7 +208,7 @@ spec:
         secretRef: s3-secret
 ```
 
-6. Apply the Custom Resource specification: `kubectl apply -f SearchHeadCluster.yaml`
+6. Apply the Custom Resource specification: ` -f SearchHeadCluster.yaml`
```
Comment (kashok-splunk) [inline docs/AppFramework.md:211]: should be `kubectl apply -n splunk-operator -f SearchHeadCluster.yaml`

Comment (vivekr-splunk) [reply docs/AppFramework.md:211]: fixed

**Review by kashok-splunk — APPROVED** [review]

**Comment by vivekr-splunk** [comment]
closing this as this was from wrong branch issues

**Pull request closed without merging**